src/components/pages/VotingPage/CastVote/CastVoteContainer.tsx

```diff
@@ -5,7 +5,9 @@ import RenderCastVotes from './RenderCastVotes';
 import { DragDropContext, DragStart, DropResult } from 'react-beautiful-dnd';
 
 const CastVoteContainer = (): React.ReactElement => {
-  const [DnDState, setDnDState] = useRecoilState(DnD.DnDContainerState);
+  const [DnDState, setDnDState] = useRecoilState<DnD.DnDContainerState>(
```

Comment:
This shouldn't be necessary once you fixed the names exports issue

Reply:
I ended up removing the exporting of the interfaces altogether